Sources/Moya/Plugins/NetworkLoggerPlugin.swift

```diff
@@ -1,135 +1,188 @@
 import Foundation
 
 /// Logs network activity (outgoing requests and incoming responses).
-public final class NetworkLoggerPlugin: PluginType {
-    fileprivate let loggerId = "Moya_Logger"
-    fileprivate let dateFormatString = "dd/MM/yyyy HH:mm:ss"
-    fileprivate let dateFormatter = DateFormatter()
-    fileprivate let separator = ", "
-    fileprivate let terminator = "\n"
-    fileprivate let cURLTerminator = "\\\n"
```

Comment:
Did we remove those constants and are using that hard coded now or they are not needed anymore?

Reply:
Previously the `dateFormatter` was set using a `dateFormat` provided in the `NetworkLoogerPlugin's` init. Now we have to directly provide the `DateFormatter`, which allow use to set it up using an hard coded `dateFormat`, or Apple's `dateStyle` and `timeStyle`. The default value uses a short date and time style.

`separator` and `terminator` are now hardCoded in the default output method.
`cURLTerminator` is no longer used.

Reply:
Nice @amaurydavid :))